### ISSUE TITLE
Signal calls are un-muted when backing out of the callscreen

### ISSUE THREAD

**Issue by Ajax471** [signalapp/Signal-Android#6859]
# Signal calls are unmuted when backing out of the callscreen

> I have:
> - [x] searched open and closed issues for duplicates
> - [x] read https://github.com/WhisperSystems/Signal-Android/wiki/Submitting-useful-bug-reports
> 
> - - - - - - - - - -
>
> #### Bug description
>
> When I mute a Signal call (and the call screen is visible at the time), if I go off of that screen (for example, by hitting the home button or locking my screen) then the call is not muted anymore and the other party can hear what's going on.
>
> #### Steps to reproduce
>
> - Open Signal
> - Make a call to another Signal member
> - Mute the call
> - Press the home button or lock your screen (call gets unmuted now)
>
> **Actual result:** Signal call becomes unmuted and the other party can hear what's going on.
>
> **Expected result:** Nothing should happen. The call should remain muted when I go out of the call screen, or if the phone's screen is locked.
>
> #### Screenshots
>
>  None that accurately show what I need to.
>
> #### Device info
>
> **Device:** LG H830 (LG-G5)
>
> **Android version:** stock Android 7.0 "(1711217429c34, NRD90U)"
>
> **Signal version:** 4.7.4
>
> #### App state
>
> Signal messages and calls registered, SMS enabled, app passphrase disabled, phone passphrase enabled 
>
> #### Link to debug log
>
> https://gist.github.com/anonymous/f560afc33d4975908e839c653ba7aa82

**Comment by wizardofid** [comment]
Are you sure about the un-muting? I just tested with my 2 devices
- Fairphone FP 1, Android 4.2.2, Signal 4.8.1
- Samsung GT-I9305 (S3 LTE), Android 4.4.4, Signal 4.9.7
On these two devices the Mute-button becomes 'unpressed', but the mic is still muted. I had to press the Mute-button two additional times to get the mic really unmuted: 
First press: the button became 'pressed' and mic was muted 
Second press: button became 'unpressed' and mic became unmuted.
Of course still a bug, but not as bad as in your case.
